services/web/src/utils/api/errors.js

```diff
@@ -2,9 +2,16 @@ import { CustomError } from 'utils/error';
 
 export class ApiError extends CustomError {
 
-  constructor(message, status) {
+  constructor(message, status, details) {
```

Comment:
don't understand the use case of these

Reply:
have a look at the forms in `/Auth/` ... that has a good use case

Reply:
sorry thats my bad :)